### DISCUSSION TITLE
Hide version selector or mike ins't working

### DISCUSSION THREAD

**Discussion by gabrielbdornas** [squidfunk/mkdocs-material#6329] [Q&A] [answered]
I'm following the [setting up versioning](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/) tutorial and its [usage](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#usage) sections says that it's best to check out [Mike's documentation](https://github.com/jimporter/mike) for further information. 

In this regard, I would like to use one of the [mike's configuration](https://github.com/jimporter/mike#configuration) to hide the version selector that is created when Mike is activated. The business rule in my case will be using versions to monitor and share improvements on my document without letting end users explicitly see it.

My `mkdocs.yml` looks like:

```yml
site_name: My Docs
theme:
  name: material
  custom_dir: overrides
extra:
  version:
    provider: mike
plugins:
  - mike:
      version_selector: false
```

But the version selector not gone:

![mkdocs-version-selector](https://user-images.githubusercontent.com/49699290/282182957-691b439f-3742-484c-81e3-853987c9d618.png)

I [asked](https://github.com/jimporter/mike/issues/151#issuecomment-1806459699) this question in one of [Mike's related issues](https://github.com/jimporter/mike/issues/151#issuecomment-1806459699) and its maintainer said that as far as he knows, "Material theme doesn't consult the `version_selector` property". It seems correct to me because when I changed the `name: material` property to `name: mkdocs` the version selector disappeared. 

Is the statement "Material theme doesn't consult the `version_selector` property" correct, or I am missing something? If yes, should I open an Issue reporting a bug or requesting a change?

Obs.: I searched for similar issues and discussions but only found questions about version selection not working, most of the time due to misconfiguration.

**Comment by squidfunk** [comment, marked as answer]
Yes, that is currently not supported by Material for MkDocs. You can open a feature request. I won't consider it a bug, as this functionality was added newly to mike, and until now, nobody cared to use it as it seems. We'll look at the feature request and check if we can support it.

**Reply by gabrielbdornas** [reply]
Hello @squidfunk, thanks for your help. As you suggested, I'll open a feature request.
